main.go

```diff
@@ -23,6 +23,8 @@ import (
 	"github.com/Southclaws/sampctl/types"
 )
 
+import _ "github.com/joho/godotenv/autoload"
```

Comment:
why is this a separate import statement?

Reply:
Sorry I was trying to work out why it wasn't working for me, and it worked like that so didn't think too much into it. I'll move it back into the import list